docs/terms-policies/governance.md

```diff
@@ -0,0 +1,240 @@
+- Onboard and mentor reviewers
+Expectations:
+- Active presence
+- Adhere to conflict of interest and bias avoidance
+Path to role:
+- Demonstrated sustained contributions and review quality
+- Nomination and consensus of existing repository Maintainers
+
+Maintainers for the respective DANDI repositories:
+| Repository | Maintainers |
+| -- | -- |
+| [dandi/dandi-archive](https://github.com/dandi/dandi-archive) | @waxlamp @jjnesbitt @mvandenburgh |
+| [dandi/dandi-cli](https://github.com/dandi/dandi-cli) | @yarikoptic @CodyCBakerPhD @candleindark |
+| [dandi/dandi-schema](https://github.com/dandi/dandi-schema) | @yarikoptic @candleindark @satra |
+| [dandi/dandi-hub](https://github.com/dandi/dandi-hub) | @asmacdo @satra |
+| [dandi/nebari-deployments](https://github.com/dandi/nebari-deployments) (Private) | @asmacdo @satra |
+| [dandi/nebari](https://github.com/dandi/nebari) | @asmacdo @satra |
+| [dandi/dandidav](https://github.com/dandi/dandidav) | @yarikoptic |
+| [dandi/dandi-about](https://github.com/dandi/dandi-about) | @kabilar @bendichter @yarikoptic @satra |
+| [dandi/dandi-docs](https://github.com/dandi/dandi-docs) | @kabilar @bendichter @yarikoptic @satra |
+| [dandi/example-notebooks](https://github.com/dandi/example-notebooks) | @kabilar @bendichter @yarikoptic @satra |
+
+### 5.4 Project Leadership
+- Current leadership team:
+    - [Satrajit Ghosh](https://satra.cogitatum.org/) ([@satra](https://github.com/satra))
+    - [Yaroslav O. Halchenko](https://centerforopenneuroscience.org/whoweare) ([@yarikoptic](https://github.com/yarikoptic))
+- Responsibilities:
+    - Approve or amend governance document and Code of Conduct
+    - Strategic project oversight
+    - Resolve escalated disputes
+    - Approve major architectural shifts
+    - Oversee risk, sustainability, funding alignment
+
+## 6. Decision-Making Model
+
+### 6.1 Roadmap
+- Project targets are discussed during the biweekly Engineering Core and Scientific Core meetings.
+- Project Leadership provides guidance on prioritization of targets.
+- Public notes of these meetings are available on [Google Drive](https://drive.google.com/drive/folders/1-jXLpcrh3L650FiZyTFgcs096nZjO2C3).
+
+### 6.2 Consensus Process
+1. Open a GitHub issue describing the bug/feature request, context, and possible solutions.
+2. For major architectural changes, create a design document and submit as a PR for discussion and refinement.
+   For reference, see [previous design documents](https://github.com/dandi/dandi-archive/tree/master/doc).
+3. Collect feedback during a 7 day review period.
+4. Summarize consensus in the GitHub issue and resolve all suggestions in the design document.
+5. Implement via pull request(s) referencing the proposed design.
+
+### 6.3 Conflict of Interest
+- Participants should disclose direct commercial interest in a technology choice.
+- Conflicted member recuses oneself from final decision phase.
+
+## 7. Pull Request Workflow
+
+### 7.1 Pull Request Requirements
```

Comment:
Given that it takes 30 days minimum to properly amend this doc, IMO we should move pull request requirements, draft-pr policy and possibly release process into the developers guide.